.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,81 @@
-<!-- 
+<!--
 Note: You may see "This branch is out-of-date with the base branch"
 when you submit a pull request. This is fine! We don't use the GitHub
 merge button to merge PRs, and you can safely ignore that message.
 
 Thanks for contributing!
 -->
 
-<!-- CHANGELOG for Downstream PRs.
+<!-- AUTOCHANGELOG for Downstream PRs.
 EXTERNAL CONTRIBUTORS: Your reviewer will most likely fill this in for you, so don't worry about this section!
 
 For some repos (currently Terraform GA/beta providers), we have the
 ability to autogenerate CHANGELOGs.
 
-Fill in the following release note code block to have it be added to the CHANGELOG, or leave the block empty if you don't expect this to be added to a downstream PR (i.e. docs-only changes or non-user facing changes)
+NO CHANGELOG NOTE: If you do not want a release note,
+please add the "changelog: no-release-note" label to this PR.
 
-Please also add any of the following appropriate labels to the PR:
-- changelog: bugfix
-- changelog: new-resource
-- changelog: new-datasource
-- changelog: deprecation
-- changelog: breaking-change
+Otherwise, fill the template out below
 -->
-# Release Note for Downstream PRs (will be copied)
-```releasenote
 
+# Release Note Template for Downstream PRs (will be copied)
+```release-note:enhancement
+
+```
+
+<!-- GUIDE FOR WRITING RELEASE NOTES
+Release notes should be formatted with one of the following headings. If
```

Comment:
If we add `release-note:note`, I think we will have covered every class of release note we *can* add that's 1:1 with a PR. If we do that, do you think we need a warning about new types here?

Reply:
It's more a description of the behavior, but I'll remove the note